streaming/base/storage/download.py

```diff
@@ -198,6 +198,37 @@ def download_from_oci(remote: str, local: str) -> None:
     os.rename(local_tmp, local)
 
 
+def download_from_r2(remote: str, local: str) -> None:
+    """Download a file from remote Cloudflare R2 to local.
+
+    Args:
+        remote (str): Remote path (S3).
```

Comment:
nit:`Remote path (S3).` -> `Remote path (R2).`

Reply:
Addressed as part of https://github.com/mosaicml/streaming/pull/255/commits/3b7e2068f524d6ff2a3b9de4908ff7064a8eb5eb.